test-integration/config.test.js

```diff
@@ -62,9 +39,5 @@ fs.readdirSync(path.resolve(__dirname, './config')).forEach(filename => {
         expect(getLog()).toMatchSnapshot();
         expect(exportToYaml(ignoreKeys(kongState))).toMatchSnapshot();
         expect(ignoreConfigOrder(getLocalState())).toEqual(kongState);
-
-        if (filename.endsWith('example.yml')) {
-            addExampleFile(configPath, filename, getLog());
```

Comment:
I don't really see any value in these example files, so I've removed it entirely. I don't feel strongly about that at all, so I'll happily restore this if someone finds it useful.

Reply:
I'm for removing it. I'm confused on what the purpose of `addExampleFile` even was. I don't see any assertions anywhere. Does it convert yml files to markdown?

Reply:
Yeah it wrote out some markdown files that included curl commands that would do the equivalent of the test. I have no idea why, but I deleted those files too!